anydesk_patch.bat

```diff
@@ -0,0 +1,138 @@
+@ECHO OFF
+
+TITLE Generate A New AnyDesk ID
+
+:: Run admin prompt
+:-------------------------------------
+REM  --> Check for permissions
+>nul 2>&1 "%SYSTEMROOT%\system32\cacls.exe" "%SYSTEMROOT%\system32\config\system"
```

Comment:
What does this line actually do 🤨? Checks for permissions for HKLM registry hive and then just sends the result to /dev/null... not very useful.

Plus, `cacls.exe` is depreciated, the new standard is `icacls.exe`.